### ISSUE TITLE
Trying to track down why Certhidea olivacea & Certhidea fusca are not sisters

### ISSUE THREAD

**Issue by hyanwong** [OpenTreeOfLife/feedback#390]
The two studies that are cited in support seem to be https://tree.opentreeoflife.org/curator/study/view/pg_2829/?tab=trees&tree=tree6579 which splits the genus, and the more recent https://tree.opentreeoflife.org/curator/study/view/ot_770/?tab=trees&tree=tree1 which supports the genus. I would hope that either the more recent study would be taken, or perhaps if there is disagreement, the one that is also supported by taxonomy. It would be useful to know from a curation POV why one study is chosen over another when there are conflicts.

================================================
Metadata   |   Do not edit below this line
:------------|:----------
Author   |   [Yan Wong](https://github.com/hyanwong) 
Upvotes   |   0 
URL   |   [tree.opentreeoflife.org/opentree/opentree9.1@ott589949/Certhidea-olivacea](https://tree.opentreeoflife.org/opentree/opentree9.1@ott589949/Certhidea-olivacea) 
Target node label   |   Certhidea olivacea 
Synthetic tree id   |   opentree9.1 
Synthetic tree node id   |   ott589949 
Source tree id(s)   |    
Open Tree Taxonomy id   |    
Supporting reference   |   N/A

**Comment by mtholder** [comment]
Yes, I'm afraid that this is cryptic to users at this point. Currently, the conflicts are resolved by the rankings of the trees in "inputs to synthesis" tree collections (see https://tree.opentreeoflife.org/curator/collections ). That is not a great way of resolving conflict, and the discoverability of the ranking is pretty low. I'll keep this issue open, as we certainly want to come up with a better way of dealing with this.

**Comment by hyanwong** [comment]
@mtholder: is there likely to be any movement on this, perchance? It's not urgent, but would be nice for it not to get lost.

**Comment by mtholder** [comment]
Hi, Sorry for the very long lag and thanks for the patience. I do think we are closer to being in feature-addition-mode. Right now @bredelings  and I are working on some scaling issues. If that goes well, then I might be able to take a look at this issue next week.

**Comment by mtholder** [comment]
I think that https://github.com/OpenTreeOfLife/collections-1/commit/fa772d44ffa2bc985f98fa6aebfc185ad5fba551 will address this on the next build, but I'll keep this open until I can check
